Engine/QuizEngineTests/GameTest.swift

```diff
@@ -9,12 +9,12 @@ import QuizEngine
 class GameTest: XCTestCase {
     
     let router = RouterSpy()
-    var game: Game<String, String, RouterSpy>!
+    lazy var game = Game(questions: ["Q1", "Q2"], router: router, correctAnswers: ["Q1": "A1", "Q2": "A2"])
```

Comment:
Like this! Was it possible before Swift 4?

Reply:
If you mean `lazy` properties, then it was.

Reply:
No, I meant using the initializer in the lazy var

Reply:
I guess, also yes.